timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/AutobatcherExecutionExceptions.java

```diff
@@ -30,7 +30,7 @@ static RuntimeException handleAutobatcherExceptions(Exception e) {
             }
             throw new RuntimeException(cause);
         } else {
-            // handle interrupted
```

Comment:
Yep, this is correct. Would it be worth having some preconditions check on the exception (that it is an `InterruptedException`)? - in case someone later on tries to re-use this and passes in some random other exception type

Reply:
sure